src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -79,10 +79,6 @@ namespace EnergyPlus {
 namespace ExhaustAirSystemManager {
     // Module containing the routines dealing with the AirLoopHVAC:ExhaustSystem
 
-    std::map<int, int> mixerIndexMap;
```

Comment:
These two globals could be part of our random unit test failures.  These could live on from the previous unit test as they aren't reset inside a state, and they are shared between threads.  Moved them to state, and this won't be a problem anymore.